<launch>

  <arg name="epochs" default="20"/>
  <arg name="batch_size" default="32"/>
  <arg name="folder" default="set1"/>
  <arg name="model" default="model1.h5"/>

  <!--
  <arg name="folder" default=""/>

  <group if="$(eval arg('folder') != '')">
  -->
    <node name="training" pkg="cnn" type="train2a.py" output="screen" required="true">
      <param name="epochs" value="$(arg epochs)"/>
      <param name="base_folder" value="$(arg folder)"/>
      <param name="modelname" value="$(arg model)"/>
      <param name="batch_size" value="$(arg batch_size)"/>
    </node>
  <!--</group>-->
  
</launch>
